### DISCUSSION TITLE
Problems with configure sd card

### DISCUSSION THREAD

**Discussion by zatolokinboris** [ReactiveCircus/android-emulator-runner#244] [General]
Hi! 
I'm not sure if this is a related question, but I'll ask anyway.

I want use emulator with sd card. And when I'll try create some emulator, I getting an error -  
Do you wish to create a custom hardware profile? [no] Error: AVD not created.
null
![image](https://user-images.githubusercontent.com/69625105/164379898-b4122384-390e-4108-8a18-528102335e6e.png)
![image](https://user-images.githubusercontent.com/69625105/164380250-e400bb73-b6f3-4ecd-8b13-c42391fc8562.png)

But in Configure emulator - it looks like that's params is accepted. 
![image](https://user-images.githubusercontent.com/69625105/164380483-6fefeecd-eaf5-44ad-bf76-d98dfd9cadc0.png)

It looks like I write incorrect params? Any ideas?

**Comment by ychescale9** [comment]
Can you share your workflow and full log?

**Reply by zatolokinboris** [reply]
Thanks for answer.

My test workflow 
![image](https://user-images.githubusercontent.com/69625105/164457043-aea7b570-aaf3-499f-ad3c-faaa9b546e8b.png)

My log arch
[logs_26.zip](https://github.com/ReactiveCircus/android-emulator-runner/files/8530990/logs_26.zip)

**Reply by zatolokinboris** [reply]
Do you have any news?

**Reply by zatolokinboris** [reply]
upd: find solution. Thanks for help

**Reply by SilverPreeceSOF** [reply]
Can you detail what the solution was? Useful for other people running into the same issue.
